docs/plugins/fetchart.rst

```diff
@@ -127,7 +127,8 @@ art::
     $ beet fetchart [-q] [query]
 
 By default the command will display all results, the ``-q`` or ``--quiet``
-switch will only display results for album arts that are still missing.
+switch will only display results for album arts that is fetched, or that is
+still missing.
```

Comment:
I think it's also not very clear what the `results` are in this context, maybe referring to the 'query` clears this up:
```
By default the command will display all albums matching the ``query``. When the ``-q`` or ``--quiet``
switch is given, only albums that were missing artwork, and whether or not it could be found, will be output.
```